compiler/damlc/daml-ide-core/dlint.yaml

```diff
@@ -477,46 +458,48 @@
 
     # MAYBE
 
-    - warn: {lhs: maybe x id, rhs: Data.Maybe.fromMaybe x}
-    - warn: {lhs: maybe False (const True), rhs: Data.Maybe.isJust}
-    - warn: {lhs: maybe True (const False), rhs: Data.Maybe.isNothing}
-    - warn: {lhs: maybe False (== x), rhs: (== Just x)}
-    - warn: {lhs: maybe True (/= x), rhs: (/= Just x)}
-    - warn: {lhs: not (isNothing x), rhs: isJust x}
-    - warn: {lhs: not (isJust x), rhs: isNothing x}
+    - warn: {lhs: maybe x identity, rhs: DA.Optional.fromOptional x}
```

Comment:
All these `maybe`s should be `optional`.

Reply:
Ok